using-timescaledb/continuous-aggregates.md

```diff
@@ -292,65 +375,14 @@ tuning other options noted below.
-an `ALTER VIEW` command like so:
-```sql
-ALTER VIEW device_summary SET (timescaledb.refresh_interval = '10 min');
-```
-Which sets the interval at which the view refreshes to 10 minutes.
-
-Other alterations to the continuous aggregate view are currently disallowed. To
-alter a continuous aggregate view in other ways it must be dropped and
-re-created; this can entail some time to re-calculate aggregations.
-
-A continuous aggregate may be dropped by using the `DROP VIEW`
-command, which deletes the hypertable that stores the materialized
-data for the continuous aggregate; it does not affect the data in the
-hypertable from which the continuous aggregate is derived (i.e., the
-raw data).  The `CASCADE` parameter is required for this command.
-
-```sql
-DROP VIEW device_summary CASCADE;
-```
-
->:WARNING: `CASCADE` will drop those objects that depend on the continuous
-aggregate, such as views that are built on top of the continuous aggregate view.
-
-
----
-
-
-### Dropping Data with Continuous Aggregates Enabled [](dropping-data)
-When dropping data in a raw hypertable using the [`drop_chunks` function][api-drop-chunks]
-that has a continuous aggregate created on it, we must specify the `cascade_to_materializations`
-argument to the `drop_chunks` call. A value of `true` will cause the continuous aggregate 
-to drop all data associated with any chunks dropped from the raw hypertable. A value
-of `false` will retain data in the continuous aggregate while dropping only the
-raw data.
-
->:TIP: When dropping data from the raw hypertable while retaining data
-on a continuous aggregate, the `older_than` parameter to `drop_chunks`
-has to be longer than the `timescaledb.ignore_invalidation_older_than`
-parameter on the continuous aggregate. That is because we cannot
-process invalidations on data regions where the raw data has been
-dropped.
-
-
-The same argument must also be supplied to the [`add_retention_policy`
-function][api-add-retention] when creating a data retention policy for a
-hypertable with a continuous aggregate.  
-
----
-
 ### Advanced Topics [](advanced-usage)
 
 #### Continuous Aggregates using Integer-Based Time [](create-integer)
 
-Usually, continuous aggregates are defined on a [date/time-type](https://www.postgresql.org/docs/current/datatype-datetime.html) column, but it is
-also possible to create your own custom scheme for handling
-aggregation for tables that are using an integer time
+Usually, continuous aggregates are defined on a
+[date/time-type](https://www.postgresql.org/docs/current/datatype-datetime.html)
+column, but it is also possible to create your own custom scheme for
+handling aggregation for tables that are using an integer time
 column. This can be useful if you have tables that use other measures
 of time that can be represented as integer values, such as nanosecond
 epochs, minutes since founding date, or whatever is suitable for your
```

Comment:
Is there a reason this example (and others) typically have a `PRIMARY KEY`?  That's usually an anti-pattern...especially setting a primary (i.e., UNIQUE) key on _time_.

Reply:
Most comes from old documentation, we can easily remove it, but why is setting a primary key on time a problem?